src/errors.js

```diff
@@ -1,11 +1,47 @@
 // src/errors
 
-const handleError = module.exports.handleError = (res, responseData) => {
-  if (!res.statusCode.toString().startsWith('2')) {
-    const apiError = new Error();
-    apiError.message = res.statusMessage;
-    apiError.code = res.statusCode;
-    apiError.data = responseData;
-    return apiError;
-  }
-};
+// TODO: test for presence of configuration errors in app JSON
+const hasError = (res, responseData) => (
```

Comment:
Is `responseData` supposed to be here? I don't see it passed in from where `hasError` is called, and `hasError` doesn't do anything with it right now. Otherwise, LGTM. 👍

Reply:
Yeah, I think that was an oversight on my part. `hasError` will eventually use it to determine if there are any app configuration errors, but that's not implemented now. I'll remove it.